amp/system-oracle/build.yml

```diff
@@ -27,5 +27,5 @@ objects:
 
 parameters:
   - name: AMP_RELEASE
-    value: 2.5.0
+    value: 2.6.0
```

Comment:
Hi,

Now the ImageStreamTags use "2.6" instead of 2.6.0 (to reflect the fact that the tag is a floating tag). This should be updated accordingly

Reply:
Thanks @hallelujah